src/baseline.rs

```diff
@@ -1,6 +1,8 @@
 pub use crate::baseline::codegen::{generate, generate_fct};
 
 mod asm;
+pub mod ast;
```

Comment:
Shouldn't we also move `baseline/expr.rs` into the ast-module?

Reply:
Yes, you're right, that would be possible.
When I think about it, the remaining content of src/baseline/info.rs could also be moved to the src/baseline/ast/info.rs, as the JitInfo is only used by the AstCodeGen.